odoo/api.py

```diff
@@ -766,6 +766,45 @@ def _lang(self):
             lang = '_' + lang
         return lang
 
+    def _(self, source: str | LazyGettext, *args, _mod='', **kwargs) -> str:
```

Comment:
This parameter name `_mod` sucks, use `_module` instead.
Also, I don't see the point of introducing it if you don't use it.  The only place you used it was useless.
And I am not even sure it is good practice: moving code from one module to another may lead to translation inconsistencies, if you don't think about checking those `_mod=...`.

Reply:
Removed, makes no sense for now.
We talked with ryv about whether transaction should depend on the module, just like what is done in JS. Still, there are context-dependent translations that we would need to handle with some parameter: today it is the module (which we also use to load only the used po files).